packages/help-center/src/tailored-section-mapping.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Do we need this file still? What happens if we just drop the beginning and ending slash and just send that with the query. Then rely on the API to decide if it exists or not.